### DISCUSSION TITLE
High RAM Usage Issue - On-disk Storage Not Taking Effect

### DISCUSSION THREAD

**Discussion by ruben-quilez** [qdrant/qdrant#4840] [General]
Thank you for your p[revious response](https://github.com/orgs/qdrant/discussions/4826) regarding the high RAM usage in my Qdrant setup. I appreciate your suggestion to use on-disk storage for vectors and indexes. However, I've encountered some issues while trying to implement this solution, and I'd like to provide more context and ask for further assistance.

### Current Situation:

I've attempted to change the configuration to use on-disk storage for both vectors and indexes using the on_disk parameter.
I've also read in the documentation that Qdrant 1.4 and later versions support updating collection parameters at runtime, including disk settings, quantization, and HNSW index configurations.
Despite these attempts, I haven't observed any changes in RAM usage or storage behavior.

### Steps Taken:

1. Modified the collection configuration to set on_disk: true for vectors and indexes.
2. Restarted the Qdrant instance and the Docker container.
3. Waited for potential background processes to complete the transition.

### Questions:

1. Why isn't the configuration change taking effect? Is there an additional step I'm missing to trigger the transition to on-disk storage?
3. Are there any logs or metrics I should check to verify if the transition process has started or encountered any issues?
4. Is there a specific API call or command I need to run to apply these changes to an existing collection?
6. Are there any potential conflicts or prerequisites that might prevent the on-disk storage from being applied?
7. Is it necessary to create the collection with on-disk storage from the beginning for this feature to work? Are there any limitations or differences in behavior when changing an existing collection to on-disk storage compared to creating it with on-disk storage initially?

Thank you for your continued support!

**Comment by generall** [comment]
Hey @ruben-quilez, have you seen https://qdrant.tech/articles/memory-consumption/ ?

**Reply by ruben-quilez** [reply]
Yes, of course. I've drawn many of my ideas from there, but it doesn't explain why my collection doesn't apply the configuration changes without having to recreate a new one
